client/src/Movies/MovieList.js

```diff
@@ -49,6 +38,6 @@ function MovieDetails({ movie }) {
           {star}
         </div>
       ))}
-    </div>
+    </div> </Link>
```

Comment:
It's preferrable to separate components/elements onto their own lines for the sake of readability:
```javascript
<Link to={`/movies/${movie.id}`}>
  <div  className="movie-card">
    <h2>{title}</h2>
    <div className="movie-director">
      Director: <em>{director}</em>
    </div>
    <div className="movie-metascore">
      Metascore: <strong>{metascore}</strong>
    </div>
    <h3>Actors</h3>
    {stars.map(star => (
      <div key={star} className="movie-star">
        {star}
      </div>
    ))}
  </div>
</Link>
```